### ISSUE TITLE
Make bookmarking useful through just the address bar.

### ISSUE THREAD

**Issue by aeschylus** [ProjectMirador/mirador#490]

**Comment by rsinghal** [comment]
We attempted to do this last fall, but found no good solution.  There would need to be a separate database that stored the current config along with the UUID in the URL.

**Comment by mejackreed** [comment]
Mirador 3 does not have the concept of bookmarking. Closing this issue.
